assets/javascripts/discourse/components/chat-user-avatar.js

```diff
@@ -3,13 +3,16 @@ import discourseComputed from "discourse-common/utils/decorators";
 import { inject as service } from "@ember/service";
 
 export default Component.extend({
-  tagName: "div",
-  classNames: ["tc-presence-flair"],
-  classNameBindings: ["online"],
+  tagName: "",
+
   chat: service(),
 
+  user: null,
+
+  avatarSize: "tiny",
+
   @discourseComputed("chat.presenceChannel.users.[]", "user.id")
-  online(users, userId) {
-    return !!users?.find((u) => u.id === userId);
+  isOnline(users, userId) {
+    return !!users?.findBy("id", userId);
```

Comment:
Ah interesting, did not know about [this](https://api.emberjs.com/ember/release/classes/EmberArray/methods/mapBy?anchor=findBy)